helm/charts/hpe-csi-driver/README.md

```diff
@@ -28,8 +28,8 @@ It's recommended to create a [values.yaml](values.yaml) file and edit it to fit
 ### Installing the Chart
 To install the chart with the name `hpe-csi`:
 ```
-helm repo add hpe https://github.com/hpe-storage/co-deployments
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system
+helm repo add hpe https://hpe-storage.github.io/co-deployments
```

Comment:
Why are we doing this?

Reply:
That is the actual repo URL, example:
```
/tmp/k8s:mmattsson$ helm repo add hpe https://github.com/hpe-storage/co-deployments
Error: Looks like "https://github.com/hpe-storage/co-deployments" is not a valid chart repository or cannot be reached: Failed to fetch https://github.com/hpe-storage/co-deployments/index.yaml : 404 Not Found
/tmp/k8s:mmattsson$ helm repo add hpe https://hpe-storage.github.io/co-deployments
"hpe" has been added to your repositories
/tmp/k8s:mmattsson$
```
Ping @shivamerla

Reply:
thanks Michael..i missed to update the URL..

Reply:
also we need to get rid of test stanza as we have removed them..